modules/videofyBidAdapter.js

```diff
@@ -250,13 +251,80 @@ function getUserSyncs(syncOptions, serverResponses) {
   }
 }
 
+function onBidWon(bid) {
+  const bidCopy = {
```

Comment:
Does this code need to be repeated 3 times? Why not have this as 1 function?
Then the export would have:
```
onBidWon: commonFunction,
onTimeout: commonFunction,
onSetTargeting: commonFunction
```